# test: fail

fn main() {
    assert(False)
}
